README.md

```diff
@@ -2397,6 +2397,45 @@ Other Style Guides
       'Batman',
       'Superman',
     ];
+    
+    // bad
+    function createHero(
+      firstName,
+      lastName,
+      inventorOf
+    ) {
+      // does nothing
+    }
+    
+    // good
+    function createHero(
+      firstName,
+      lastName,
+      inventorOf,
+    ) {
+      // does nothing
+    }
+    
+    // good
+    function createHero(firstName,
+                        lastName,
```

Comment:
I've marked this branch as editable by maintainers. Otherwise you can wait a few hours for me to get to a keyboard